csharp-basic-topics/DetectKeyPressInCSharp/DetectKeyPressInCSharp/DetectKeyPressInCSharp/ConsoleApp.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace DetectKeyPressInCSharp
+{
+    public class ConsoleApp
+    {
+        private readonly IConsoleService _consoleService;
+        public ConsoleApp(IConsoleService consoleService)
```

Comment:
@mohitkumar342 
There should be a space here

Reply:
Done